### ISSUE TITLE
"Bugfix Validation" task is broken

### ISSUE THREAD

**Issue by Algunenano** [ClickHouse/ClickHouse#65946]
It seems it has been broken for a few weeks.

Example: https://s3.amazonaws.com/clickhouse-test-reports/65835/d627709179db7b3af9b6e2f1b1c30b3e87da0cb1/bugfix_validation.html

It doesn't run anything so then it complains about `[with NOT_OK] Nothing to run	FAIL`

```
++ rg -v '#' /usr/share/zoneinfo/zone.tab
++ head -n1
++ shuf
++ awk '{print $3}'
+ TZ=America/Argentina/Buenos_Aires
+ echo 'Choosen random timezone America/Argentina/Buenos_Aires'
Choosen random timezone America/Argentina/Buenos_Aires
+ ln -snf /usr/share/zoneinfo/America/Argentina/Buenos_Aires /etc/localtime
+ echo America/Argentina/Buenos_Aires
+ dpkg -i package_folder/clickhouse-common-static_24.6.1.4423_amd64.deb
Selecting previously unselected package clickhouse-common-static.
(Reading database ... 48846 files and directories currently installed.)
Preparing to unpack .../clickhouse-common-static_24.6.1.4423_amd64.deb ...
Unpacking clickhouse-common-static (24.6.1.4423) ...
Setting up clickhouse-common-static (24.6.1.4423) ...
+ dpkg -i 'package_folder/clickhouse-common-static-dbg_*.deb'
dpkg: error: cannot access archive 'package_folder/clickhouse-common-static-dbg_*.deb': No such file or directory
```

**Comment by pamarcos** [comment]
Another one: https://s3.amazonaws.com/clickhouse-test-reports/65838/389a86ec059461f20521e4dd2c7888e2b3b37623/bugfix_validation/run__stateless_tests__release_.log

**Comment by Algunenano** [comment]
Seems it was  broken by https://github.com/ClickHouse/ClickHouse/pull/64848

**Comment by Algunenano** [comment]
Should be fixed by https://github.com/ClickHouse/ClickHouse/issues/65946

**Comment by pamarcos** [comment]
> Should be fixed by #65946

I think you meant https://github.com/ClickHouse/ClickHouse/pull/66120/
